<ui version="4.0" >
 <class>UserHistoryCfgBase</class>
 <widget class="QWidget" name="UserHistoryCfgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string></string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QSpinBox" name="edtSize" >
     <property name="maximum" >
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLabel" name="lblSize1" >
     <property name="text" >
      <string>Mb</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QCheckBox" name="chkSize" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="lblSize" >
     <property name="text" >
      <string>Max history file size</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="edtDays" >
     <property name="maximum" >
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="lblDays1" >
     <property name="text" >
      <string>days</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="lblDays" >
     <property name="text" >
      <string>Keep history </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="chkDays" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
